pom.xml

```diff
@@ -145,6 +145,34 @@
 			</exclusions>
 		</dependency>
 
+		<dependency>
+			<groupId>eu.europeana.metis</groupId>
+			<artifactId>metis-indexing</artifactId>
+			<version>${version.metis}</version>
+			<exclusions>
+				<exclusion>
+					<groupId>org.apache.logging.log4j</groupId>
+					<artifactId>log4j-slf4j-impl</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+
+		<dependency>
+			<groupId>eu.europeana.corelib</groupId>
```

Comment:
Took me time to figure but this one is required by indexing, but the enrich pom explicitly excludes it, so I had to add it again